container/root/etc/services-available/php-fpm/run

```diff
@@ -8,22 +8,11 @@ trap
 {
   term
   {
-    foreground
-    {
-      backtick -i -n NGINXPID { cat /tmp/.nginx/nginx.pid }
-    }
-    import -u NGINXPID ifthenelse { $NGINXPID }
-    {
-      foreground
-      {
-        echo [sigterm-phpfpm] waiting for nginx to exit
-      }
-      wait { $NGINXPID } import -u ! kill -s SIGQUIT ${!}
-    }
-    foreground
-    {
-      import -u ! kill -s SIGQUIT ${!}
-    }
+    echo [run php-fpm] TERM received
```

Comment:
these are just echoing now and not actually doing anything?

Reply:
bingo. we're suppressing these signals: 
https://github.com/behance/docker-base#implementing-graceful-shutdown-from-admin-initiated-container-termination